packages/grid/src/vaadin-grid-selection-column-base-mixin.js

```diff
@@ -60,6 +61,17 @@ export const GridSelectionColumnBaseMixin = (superClass) =>
           value: false,
         },
 
+        /**
+         * When true, rows can be selected by dragging mouse cursor over selection column.
+         * @attr {boolean} drag-select
+         * @type {boolean}
+         */
+        dragSelect: {
+          type: Boolean,
+          value: false,
+          reflectToAttribute: true,
```

Comment:
nit: reflecting the property as an attribute shouldn't be necessary

Reply:
Done